src/dnf-plugins/product-id.py

```diff
@@ -20,9 +20,10 @@
 from subscription_manager.productid import ProductManager
 from subscription_manager.utils import chroot
 from subscription_manager.injectioninit import init_dep_injection
+from subscription_manager.i18n import ugettext as _
```

Comment:
Please completely remove usage of `_()` in the product-id plugin, because `_()` is used for translation of log message. Log message should never been translated.

Reply:
Please note that `logger` is `dnfpluginscore.logger`, not Python's `logging`. Also, I checked in the sources of dnf-plugins-core, and it seems that the vast majority of all the `dnfpluginscore.logger` usage with priority `info()` and greater (`warning()` and `error()`) are translated. Also, it seems like `info()` messages (and greater) are shown by default.

Reply:
Oh, they translate log messages. Interesting.